tiles-generation/cdk/bin/cdk.ts

```diff
@@ -11,110 +11,167 @@ import { App } from 'aws-cdk-lib';
+        process.env.CDK_DEFAULT_REGION,
+    account:
+        app.node.tryGetContext('account') ||
+        process.env.CDK_INTEG_ACCOUNT ||
+        process.env.CDK_DEFAULT_ACCOUNT,
 };
 
-const testInstanceType = ec2.InstanceType.of(ec2.InstanceClass.MEMORY3, ec2.InstanceSize.XLARGE);
+const testInstanceType = ec2.InstanceType.of(
+    ec2.InstanceClass.MEMORY3,
+    ec2.InstanceSize.XLARGE
+);
+
+const PlanetInstanceType = ec2.InstanceType.of(
+    ec2.InstanceClass.X1E,
+    ec2.InstanceSize.XLARGE8
+);
 
 // Define volume size for test stack instance
 const testEC2Volume = asc.BlockDeviceVolume.ebs(30, {
-  deleteOnTermination: true,
-  // https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html
-  volumeType: asc.EbsDeviceVolumeType.IO1,
-  // The number of I/O operations per second (IOPS) to provision for the volume
-  iops: 1000
-});
-
-//Define docker environment for test stack ecs task
-const testDockerEnv = {
-  // Bounding box defines the area that tiles will be generated, it's the area for luxembourg
-  'BBOX':'5.654568,49.431689,6.574673,50.177945',
-  // Maps raw data 
-  'DOWNLOAD_PBF':'https://download.geofabrik.de/europe/luxembourg-latest.osm.pbf',
-  // Configuration for OSM2OGSQL https://osm2pgsql.org/doc/manual.html
-  'OSM2PGSQL_EXTRA_ARGS':'-C 10000',
-  // Minimum zoom level for generation tiles
-  'MIN_ZOOM':'0',
-  // Maximum level for generation tiles
-  'MAX_ZOOM':'10',
-  // Threads for import and rendering tiles process.
-  'THREADS':'8',
-  // Make generate_tiles.py logs output to container log
-  'PYTHONUNBUFFERED':'1',
-  // The database use the autovacuum feature by default. This behavior can be changed with AUTOVACUUM environment variable
-  'AUTOVACUUM':'off'
-};
-
-const testTileGenerationStack = new TileGenerationStack(app, 'TestTileGenerationStack', {    
-  env: env,
-  instanceType: testInstanceType,
-  volume: testEC2Volume,
-  // The value for the size (in MiB) of the /dev/shm volume for container default 
-  // Default limit 64MB is too low for container
-  sharedMemorySize: 200,
-  // Docker environment for test stack ecs task
-  dockerEnv: testDockerEnv,
-  // Memory reservation for the task.
-  memoryReservationMiB: 25000
+    deleteOnTermination: true,
+    // https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html
+    volumeType: asc.EbsDeviceVolumeType.IO1,
+    // The number of I/O operations per second (IOPS) to provision for the volume
+    iops: 1000,
 });
 
-const PlanetInstanceType = ec2.InstanceType.of(ec2.InstanceClass.X1E, ec2.InstanceSize.XLARGE8);
-
 // For planet tile generation, the whole process maximum volume will reach to about 1300GB
-const PlanetEC2Volume = asc.BlockDeviceVolume.ebs(1500, {
-  deleteOnTermination: true,
-  volumeType: asc.EbsDeviceVolumeType.IO1,
-  iops: 5000
+const planetEC2Volume = asc.BlockDeviceVolume.ebs(1500, {
+    deleteOnTermination: true,
+    volumeType: asc.EbsDeviceVolumeType.IO1,
+    iops: 5000,
```

Comment:
are we using tabs instead of spaces here?

Reply:
there should be a way to convert the tabs to spaces or not have the tabs. Can you please validate that,

Reply:
On the prettier official docs, the `tabWidth` is used to define the spaces, so here is  4 spaces per indentation-level. https://prettier.io/docs/en/options.html#tab-width